### PR TITLE
Blog post - Build a Navbar Mega Menu Using React Bootstrap

### PR DESCRIPTION
**Pull request by teallarson**

### CODE REVIEW

File: pages/blog/build-a-mega-menu-with-react-bootstrap.mdx [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+  src="build-a-mega-menu-with-react-bootstrap/mockups-screenshot.png"
+  width={845}
+  height={251}
+/>
+
+We're going to build something similar where the "events" link currently lives on our template.
+
+Let's start by adding the following into `/components/navigation.js` in place of the current `Nav.Link` for events:
+
+```
```
Comment (evantahler) [inline pages/blog/build-a-mega-menu-with-react-bootstrap.mdx]: try ```tsx so the code gets styles!

File: pages/blog/build-a-mega-menu-with-react-bootstrap.mdx [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+
+<Image
+  centered
+  alt="Copycat Navbar we'll be making"
+  src="build-a-mega-menu-with-react-bootstrap/finished-tutorial.png"
+  width={845}
+  height={269}
+/>
+
+Coffee themed, because of course. I've created a [basic boilerplate](https://github.com/grouparoo/nextjs-react-bootstrap-boilerplate)
```
Comment (evantahler) [inline pages/blog/build-a-mega-menu-with-react-bootstrap.mdx]: If you want to highlight the boilerplate repo, you could use a `<Button>` or `<Alert>` (yay mdx!)

**Review by evantahler — APPROVED** [review]
Awesome!

File: pages/blog/build-a-mega-menu-with-react-bootstrap.mdx (hunk 1 above)
Comment (tealjulia) [reply pages/blog/build-a-mega-menu-with-react-bootstrap.mdx:76]: To clarify, you mean just putting "tsx" after the three backticks?  Or converting the project to tsx?

**Pull request merged**